### PR TITLE
Updated references to master

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
Check list:
- [x] Add an entry to the [changelog](/CHANGELOG.md)
- [ ] [Squash all commits into a single one](/CONTRIBUTING.md)
- [x] Describe your PR, link issues, etc.

This PR will resolve issue #381 .

### CODE REVIEW

**Review by etagwerker — CHANGES_REQUESTED** [review]
@96RadhikaJadhav Hi there, thanks for getting this started! 

I see that the default branch for a compare and mode-ci action has changed, but I can't find the change in the set of changes that you submitted. 

It seems that you are missing updates in these files: 

- https://github.com/whitesmith/rubycritic/blob/d673a2c75e4fb1d7f6776b77c63401a4aedf0112/lib/rubycritic/cli/options/argv.rb#L79-L84
- https://github.com/whitesmith/rubycritic/blob/d673a2c75e4fb1d7f6776b77c63401a4aedf0112/lib/rubycritic/cli/options/file.rb#L40-L44

File: CHANGELOG.md [hunk 1]
```diff
@@ -357,3 +357,4 @@
 [@Flink]: https://github.com/Flink
 [@jackcasey]: https://github.com/jackcasey
 [@sl4vr]: https://github.com/sl4vr
+[@96RadhikaJadhav]: https://github.com/96RadhikaJadhav/
```
Comment (nunosilva800) [inline CHANGELOG.md:360]: Don't forget to add a 1-line summary of your change on the top of the file :)

Comment (96RadhikaJadhav) [reply CHANGELOG.md:364]: @nunosilva800  Done.

Comment (nunosilva800) [reply CHANGELOG.md:364]: I toke the liberty to bring in changes from upstream and fix the conflicts.
You can pull to get the latest changes on your machine.

https://github.com/whitesmith/rubycritic/pull/384/commits/a75366fb866ca2324a717d247056432c22239f25

**Review by nunosilva800 — APPROVED** [review]
Even though travis does not feel like working now, locally all tests run.

This LGTM! Well done @96RadhikaJadhav thanks!

File: lib/rubycritic/generators/html/templates/code_index.html.erb [hunk 2]
```diff
@@ -24,7 +24,7 @@
               <% unless Config.suppress_ratings %>
                 <td class="center">
                   <% if Config.build_mode? %>
-                    <% master_analysed_module = Config.base_branch_collection.find(analysed_module.pathname) %>
+                    <% master_analysed_module = Config.base__collection.find(analysed_module.pathname) %>
```
Comment (etagwerker) [inline lib/rubycritic/generators/html/templates/code_index.html.erb:27]: @96RadhikaJadhav This looks like a typo. Is this necessary?

**Review by etagwerker — CHANGES_REQUESTED** [review]
@96RadhikaJadhav I only noticed one issue. Could you please look into my comment? Thanks!

**Comment by nunosilva800** [comment]
@96RadhikaJadhav will you be continuing this?

**Comment by faisal** [comment]
Trying to complete this -- or at least move it farther along -- in https://github.com/whitesmith/rubycritic/pull/444.

**Pull request closed without merging**